src/utils/fs.spec.js

```diff
@@ -151,6 +151,29 @@ describe('findUpwardsFile()', () => {
   })
 })
 describe('listDirectoryFiles()', () => {
-  it.todo('lists files in the supplied directory')
-  it.todo('???')
+  it('lists files in the supplied directory', async () => {
+    const { listDirectoryFiles } = jest.requireActual('./fs')
+    const { Dirent, constants } = jest.requireActual('fs')
+    const { UV_DIRENT_FILE } = constants
+
+    const file = new Dirent('file', UV_DIRENT_FILE)
+
+    fs.readdir.mockResolvedValue([ file ])
+
+    const files = await listDirectoryFiles()
+
+    expect(files).toMatchObject([ file.name ])
+  })
+  it('ignores subdirectories in the supplied directory', async () => {
+    const { listDirectoryFiles } = jest.requireActual('./fs')
+    const { Dirent, constants } = jest.requireActual('fs')
+    const { UV_DIRENT_DIR } = constants
+
+    const directory = new Dirent('directory', UV_DIRENT_DIR)
```

Comment:
I'm trying to read up on the constructor arguments for Dirent but I can't seem to find anything about it. Also, `UV_DIRENT_DIR` seems to be an undocumented constant; where'd you find out about this pattern? 🗺️ ❔

Reply:
Found the same usage pattern in Node's internal tests for fs, but it required some digging to get there 😄 
https://github.com/nodejs/node/blob/c17dcb32533aa007dfbf507d22c28ef3c7c11c29/test/parallel/test-fs-readdir-types.js#L114

Still curious on how you found about it 😮

Reply:
Just went through the code in the nodejs github repo. =)